docs/releasing.md

```diff
@@ -0,0 +1,38 @@
+# Releasing Flintlock
+
+> These instructions will change when we start supporting previous versions whilst using main for future versions.
+
+## Determine release version
+
+The projects follows [semantic versioning](https://semver.org/#semantic-versioning-200) and so the release version must adhere to this specification. Depending on the changes in the release you will need to decide the next appropriate version number.
+
+## Create tag
+
+* Checkout upstream main
+* Create a tag with the version number:
+
+```bash
+RELEASE_VERSION=v0.1.0-alpha.1
+git tag -s ${RELEASE_VERSION} -m "${RELEASE_VERSION}
```

Comment:
```suggestion
git tag -s "${RELEASE_VERSION}" -m "${RELEASE_VERSION}"
```

Reply:
The first two `"` is optional, but the last one is important, because `-m "${RELEASE_VERSION}` would just wait on the next line.

Reply:
Doh, typo. Changed.